BMM.UI.Android/Resources/layout/listitem_fra_kaare_teaser.axml

```diff
@@ -1,150 +1,142 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
We used the CardView before because it allows us to use rounded corners. Is the new solution less flexible regarding screen sizes?